src/case-monitoring/supplier-utils.ts

```diff
@@ -221,6 +221,15 @@ export class ProcessExecutor {
         });
       logProcessExecution('DONE call execution of next step', executionStep.nextExecutionStep);
     } else if (executionStep.isLastStep) {
+      // Display notification
+      const notyf = configureToast(3000);
+      // Event_13tn0ty is the abort event
+      if (executionStep.id === 'Event_13tn0ty') {
+        toast(notyf, 'The email is aborted');
```

Comment:
**suggestion:** Can we change the message to "The process has been aborted. No email sent"?